.travis.yml

```diff
@@ -0,0 +1,3 @@
+language: ruby
```

Comment:
`sudo: false` will make it run faster

http://docs.travis-ci.com/user/workers/container-based-infrastructure/